code/modules/projectiles/guns/energy/special.dm

```diff
@@ -79,6 +79,7 @@
 	name = "\improper Mind Flayer"
 	desc = "A prototype weapon recovered from the ruins of Research-Station Epsilon."
 	icon_state = "xray"
+	item_state = null

```

Comment:
As I told kor, this is dumb but needed; https://github.com/tgstation/-tg-station/blob/master/code/modules/projectiles/guns/energy/nuclear.dm